src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VilkårTabellRad.tsx

```diff
@@ -120,7 +127,13 @@ const VilkårTabellRad: React.FC<IProps> = ({
                 </BodyShort>
             </Table.DataCell>
             <Table.DataCell>
-                <BeskrivelseCelle children={vilkårResultat.verdi.begrunnelse.verdi} />
+                {vilkårResultat.verdi.begrunnelse.verdi ? (
+                    <StyleTooltip content={vilkårResultat.verdi.begrunnelse.verdi}>
+                        <BeskrivelseCelle children={vilkårResultat.verdi.begrunnelse.verdi} />
+                    </StyleTooltip>
+                ) : (
+                    <BeskrivelseCelle children={vilkårResultat.verdi.begrunnelse.verdi} />
+                )}
```

Comment:
Tror ikke vi trenger en `else` her. Tenker det er ryddigere å ikke tegne ut noe som helst dersom `vilkårResultat.verdi.begrunnelse.verdi` ikke er satt.
```suggestion
                {vilkårResultat.verdi.begrunnelse.verdi && (
                    <StyleTooltip content={vilkårResultat.verdi.begrunnelse.verdi}>
                        <BeskrivelseCelle children={vilkårResultat.verdi.begrunnelse.verdi} />
                    </StyleTooltip>
                )}
```